CHANGELOG.md

```diff
@@ -1,41 +1,81 @@
 ## 0.3.0
 
-- Node >=10.x now required.
-- Added getElementsByClassName method.
-- Added Node to the list of exports
-- Added lowercase of åäö in entityMap.
-- Moved existing sources into `lib` subdirectory.
-- Removed `.npmignore` in favor of `files` entry in package.json.
-- More `package.json` refactoring.
-- Replaced CHANGELOG with more rigorous file.
-- Replaced LICENSE with more rigorous file.
-- Removed component.json (deprecated package manager https://github.com/componentjs/guide)
-- `proof` devDep updated to latest.
-- Fixed CI.
+[Commits](https://github.com/xmldom/xmldom/compare/0.2.1...0.3.0)
+
+- **BREAKING** Node >=10.x is now required.
+- **BREAKING** Removed component.json (deprecated package manager https://github.com/componentjs/guide)
+- **BREAKING** Moved existing sources into `lib` subdirectory.
+- Update devDependency `proof`
+- Removed `.npmignore` in favor of `files` entry in `package.json`.
```

Comment:
Is removing `. npmignore ` visible from the npm package?

(I think it is but am not 100% sure. I do recall npm always converting `.gitignore` to `.npmignore`.)

I would maybe turn this around:

```suggestion
- Introduce `files` entry in `package.json` and remove use of `.npmignore`.
```